src/mixins/filing-template-mixin.ts

```diff
@@ -256,16 +256,18 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
           email: this.getBusinessContact.email,
           phone: this.getBusinessContact.phone,
           extension: this.getBusinessContact.extension
-        },
-        courtOrder: {
-          fileNumber: this.getFileNumber
         }
       }
     }
 
-    // Save this boolean to drafts only
-    // It is a state required for UX when resuming drafts
-    if (isDraft) filing.alteration.courtOrder.hasPlanOfArrangement = this.getHasPlanOfArrangement
+    // Apply Court Order ONLY when it is required and applied
+    if (this.getHasPlanOfArrangement || this.getFileNumber) {
+      filing.alteration.courtOrder = {
+        fileNumber: this.getFileNumber,
+        effectOfOrder: EffectOfOrders.PLAN_OF_ARRANGEMENT,
```

Comment:
so I don't believe that `effectOfOrder` should always have a value of `planOfArrangement`.  For example, if the user only provides the court order number and leaves the plan of arrangement checkbox unchecked, I believe we should be passing a null value for `effectOfOrder`

Reply:
i don't know anything about this properties value, what other options there could be potentially etc. 
In the current state, this is the only value this property would ever have though 🤔 

I'm ok with modifying this later if we need to make it dynamic but until the time when we have some variance, i feel like it's ok to leave it, because i'm not even sure IF we will have that variance in this UI.

Reply:
Ah i do see what you mean though. Using the checkbox as this particular properties definer.

Reply:
i could refactor this to conditional apply the enum value, i'll see how the api handles a null value 🤞

Reply:
Maybe I'm missing something here.  If you pass in `planOfArrangement` as the value for `filing.alteration.courtOrder.effectOfOrder` when only the court order number has been provided via the UI, then how would you be able to distinguish when the plan of arrangement checkbox should be unchecked?  The api will come back with the an effectOfOrder value of `planOfArrangement` when you retrieve that alteration the next time.

Reply:
The api seems to accept null and an empty string for `effectOfOrder` when I tested it via postman.

Reply:
I think you mean via a draft?
I'll ping you!

Reply:
Updated as per comments 👍

Reply:
took a look at the addition of the condition apply enum value.  Looks good!  👍